Lesson 13/src/index.js

```diff
@@ -0,0 +1,131 @@
+// 1
+const nestedArray = [[1, 2], [3, [4, 5]], 6]
+const flatArray = nestedArray.flat(Infinity)
+
+console.log(flatArray)
+
+// 2
+
+const products = [
+  { id: 1, name: 'Телефон', inStock: true },
+  { id: 2, name: 'Ноутбук', inStock: false },
+  { id: 3, name: 'Часы', inStock: false }
+]
+
+function productInStock(productsArray) {
+  return productsArray.some((product) => product.inStock === true)
+}
+
+console.log(productInStock(products))
```

Comment:
```suggestion
function isSomeProductInStock(productsArray) {
  return productsArray.some((product) => product.inStock)
}

console.log(isSomeProductInStock(products))
```
1. inStock имеет значение true/false, можно сразу возвращать как в результат some (не нужно явно указывать === true)

2. по наименованию функции, мы продолжаем стараться начинать название с глагола и указывать на ответ

для функций, которые возвращают булево значение, можно начинать название с `is` - такие названия можно часто встретить, я тоже использую когда глагола лучше не находится

пример `isSomeProductInStock` - понятно, что проверяется и ответ